docs/src/man/multistart_formulation.md

```diff
@@ -0,0 +1,200 @@
+# Power Grid Lib - Unit Commitment / Multi-Start Unit Commitment
+This formulation is from the benchmark library maintained by the IEEE PES Task Force on Benchmarks for Validation of Emerging Power System Algorithms and is designed to evaluate a well established version of the the Unit Commitment problem.
+
+# Formulation Overview
+ The features of this model are:
+- A global load requirement with time series
+- An optional global spinning reserve requirement with time series
+- Thermal generators with technical parameters, including
+  - Minimum and maximum power output
+  - Hourly ramp-up and ramp-down rates
+  - Start-up and shut-down ramp rates
+  - Minimum run-times and off-times
+  - Upto 3 types of startup types (hot, warm, cold)
+  - Off time dependent start-up costs
+  - Startup & Shutdown lag/power trajectory constraint
+  - Piecewise linear convex production costs
+  - Must run constraints
+  - No-load costs
+- Optional renewable generators with time series for minimum and maximum production.
+
+
+# Formultions 
+A detailed description of this mathematical model is available here. This model does have some aaugmentation to constraints but is mathematically equivalent to the formulation found [here](https://github.com/power-grid-lib/pglib-uc/blob/master/MODEL.pdf). 
+### Indices and Sets
+```math
+\begin{itemize}
+	\item[$g \in \cG$] Set of thermal generators.
+	\item[$g \in \cG_{\textit{on}}^0$] Set of thermal generators which are initially committed (on).
+	\item[$g \in \cG_{\textit{off}}^0$] Set of thermal generators which are not initially committed (off).
+	\item[$w \in \cW$] Set of renewable generators.
+	\item[$t \in \cT$] Hourly time steps: $1, \ldots, T$, $T$ = {\tt time\_periods}
+	\item[$l \in \cL_g$] Piecewise production cost intervals for thermal generator $g$: $1, \ldots, L_g$.
+	\item[$s \in \cS_g$] Startup categories for thermal generator $g$, from hottest ($1$) to coldest ($S_g$): $1, \ldots, S_g$.
+\end{itemize}
+```
+
+### System Parameters
+```math
+\begin{itemize}
```

Comment:
These are pieces of the formulation that pertain to the whole system. We should keep only the ThermalGeneration constraints here and use the references to the other repo if someone is interested in the whole UC.

Reply:
I'll keep the parts related to Thermal UC